packages/core-components/api-report-alpha.md

```diff
@@ -0,0 +1,73 @@
+## API Report File for "@backstage/core-components"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { TranslationRef } from '@backstage/core-plugin-api/alpha';
+
+// @alpha (undocumented)
+export const coreComponentsTranslationRef: TranslationRef<
+  'core-components',
+  {
+    readonly 'link.openNewWindow': 'Opens in a new window';
+    readonly 'table.filter.title': 'Filters';
+    readonly 'table.filter.clearAll': 'Clear all';
+    readonly 'signIn.title': 'Sign In';
+    readonly 'signIn.loginFailed': 'Login failed';
+    readonly 'signIn.customProvider.title': 'Custom User';
+    readonly 'signIn.customProvider.subtitle2': 'This selection will not be stored.';
+    readonly 'signIn.customProvider.subtitle': 'Enter your own User ID and credentials.';
```

Comment:
Think that we should condense these keys if possible with a `\n` or `<br/>` in the middle. Think that these should be one message rather than splitting it up into two keys.

Reply:
done